bin/run-tests.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -xe
 
 flake8
+moz-l10n-lint l10n/l10n-pontoon.toml
+moz-l10n-lint l10n/l10n-vendor.toml
 python manage.py lint_ftl -q
```

Comment:
do we still need this command?

Reply:
I don't know, does `lint_ftl` run against the files pulled from the external repo? If that's the case, I'd say let's keep them? This PR adds linting mainly for the `en` files that live in bedrock.